orttraining/orttraining/test/python/orttraining_test_orttrainer_frontend.py

```diff
@@ -711,6 +711,39 @@ def testORTTrainerMixedPrecisionLossScaler(seed, device, expected_loss, fetches)
     assert trainer._onnx_model is not None
 
 
+@pytest.mark.parametrize("seed,device,gradient_accumulation_steps,total_steps,expected_loss", [
+    (0, 'cuda', 1, 12, [10.5368022919, 10.4146203995, 10.3635568619, 10.2650547028, 10.2284049988, 10.1304626465,\
```

Comment:
Just curious - where do we get these hard numbers from?

Reply:
From the current API implementation. I assume the current API is correct and the new frontend has to match exactly
There is a test to compare legacy and experimental too